blocks/eda-frontend/src/components/SchematicEditor/ComponentProperties.js

```diff
@@ -225,15 +305,19 @@ export default function ComponentProperties () {
             const rootKeyName = keyName.substr(0, 4)
             const typeId = rootKeyName + '_type'
             const helpId = rootKeyName + '_help'
-            if (compProperties !== undefined && compProperties[rootKeyName] !== null && compProperties[typeId] !== null) {
+            const compType = compProperties && compProperties[typeId]
+            const compHelp = compProperties && compProperties[helpId]
+            const helperText = errorFields[keyName]
+              ? getHelperText(compType)
+              : compHelp
+            if (compProperties && compProperties[rootKeyName] !== null && compType !== null && val[keyName] !== null) {
               return (
                 <ListItem key={i}>
-                  <TextField id={keyName} label={compProperties[rootKeyName]} value={val[keyName] || ''} helperText={compProperties[helpId]} size='small' variant='outlined' onChange={getInputValues} />
+                  <TextField id={keyName} label={compProperties[rootKeyName]} value={val[keyName] || ''} helperText={helperText} error={errorFields[keyName]} size='small' variant='outlined' onChange={getInputValues} />
```

Comment:
```suggestion
                  <TextField id={keyName} label={compProperties[rootKeyName]} value={val[keyName] || ''} helperText={helperText} error={error} size='small' variant='outlined' onChange={getInputValues} />
```
Assign to error first like this. Then use it like above.

`const error = errorrFields[keyName] !== '' ? true : false`